frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1135,10 +1136,11 @@ function ReaderRolling:showEngineProgress(percent)
             self.engine_progress_update_not_before = now + ENGINE_PROGRESS_INITIAL_DELAY
             return
         end
+
         -- Widget size and position: best to anchor it at top left,
         -- so it does not override the footer or a bookmark dogear
         local x = 0
-        local y = Size.margin.small
+        local y = Size.margin.small + (self.ui.document.been_rendered and self.ui.document:getHeaderHeight() or 0)
```

Comment:
That easy ! :)
May be just make that add to `y` in a `if self.ui.document.been_rendered then` to make it clearer, with a comment
`-- Show it below CRE top status bar if there is one showing`.
Or may be just replace y if there is a header, the `Size.margin.small` was just there for when at top of screen, it might be covered by the bezel (depending on the angle you look at the device, when making dinner while it's re-rendering :) on some devices - you may not need this gab when below the status bar (or may be 1px if needed ?).